io/src/test/scala/fs2/io/IoSuite.scala

```diff
@@ -147,6 +147,19 @@ class IoSuite extends Fs2Suite {
         }
       }
     }
+
+    test("PipedInput/OutputStream used to track threads, fs2 reimplementation works") {
```

Comment:
This was adapted for CE2; it was originally:

```scala
    test("PipedInput/OutputStream used to track threads, fs2 reimplementation works") {
      readOutputStream(1024) { os =>
        IO.blocking {
          val t = new Thread(() => os.write(123))
          t.start
          t.join
          Thread.sleep(100L)
        }
      }.compile.drain.map(_ => assert(true))
    }
```

Reply:
I'll take a look when I'm back in front of my computer a little later.